backend/danswer/background/celery/versioned_apps/heavy.py

```diff
@@ -0,0 +1,15 @@
+"""Factory stub for running celery worker / celery beat."""
+from celery import Celery
+
+from danswer.utils.variable_functionality import set_is_ee_based_on_env_variable
+
+set_is_ee_based_on_env_variable()
+
+
+def get_app() -> Celery:
```

Comment:
small: can we add a comment as to why these are different from `beat.py`? And a comment about how we need to make a change to these to match `beat.py` if we want to add `ee` specific tasks?